modules/living_spaces_users/src/Plugin/Block/LivingSpacesUsersMasqueradeBlock.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\living_spaces_users\Plugin\Block;
+
+use Drupal\masquerade\Plugin\Block\MasqueradeBlock;
+use Drupal\masquerade\Form\MasqueradeForm;
```

Comment:
The module references masquerade classes, but does not have dependency in info.yml